krypto/src/commonMain/kotlin/com/soywiz/krypto/encoding/Base64url.kt

```diff
@@ -0,0 +1,41 @@
+package com.soywiz.krypto.encoding
+
+/**
+ * Base64Url codec based on the Base64 codec and RFC 7515 Appendix C.
+ */
+object Base64Url {
+    operator fun invoke(v: String) = decodeIgnoringSpaces(v)
+    operator fun invoke(v: ByteArray) = encode(v)
+
+    fun decode(str: String): ByteArray {
+        var s = str.replace("-", "+").replace("_", "/")
+        when (s.length % 4) {
+            0 -> { }
+            2 -> s += "=="
+            3 -> s += "="
+            else -> throw IllegalArgumentException("Illegal base64url string")
+        }
+
+        return Base64.decode(s)
+    }
+
+    fun decodeIgnoringSpaces(str: String): ByteArray {
+        return decode(str.replace(" ", "").replace("\n", "").replace("\r", "").replace("\ts", ""))
```

Comment:
Probably \ts should be \t

Reply:
I copied this directly from Base64.decodeIgnoreingSpaces, where it also has "\ts", and didn't notice this. But I agree it should probably be "\t", in both places.

Reply:
Oh fu :) then we had a bug, and something to test